### DISCUSSION TITLE
Sensors calibrating

### DISCUSSION THREAD

**Discussion by Slattfpv** [iNavFlight/inav#8097] [Q&A] [answered]
Hey guys I’m having a problem using the holybro h7 with inav 4.1.0. Same settings and dump on multiple fcs and some give me this problem where the screen says “sensors calibrating” and I can never get past that warning. If I plug in the usb first and connect to inav and then plug in the battery it will then work no problem and fly great. Very frustrating and I can’t find any leads. Thanks in advance!

**Comment by Jetrell** [comment, marked as answer]
The craft needs to be totally motionless for a few seconds after the FC is powered, to calibrate the IMU.

iNav 5.0 has a feature - `init_gyro_cal = off `which allows initial calibration of the FC on a moving vehicle, like a boat.
I wouldn't recommend using it all the time. But you could load 5.0 and try this setting. To see if it helps, in case its a hardware issue.
